src/test/tests.cpp

```diff
@@ -111,52 +111,50 @@ TEST_CASE("replace chars not in set", "[utils]") {
 	}
 }
 
-TEST_CASE("separate fully qualified name into style and name", "[utils]") {
-	auto testCall = [](const std::wstring& input, std::wstring& style, std::wstring& name) {
+TEST_CASE("tokenize string at first token", "[utils]") {
+	auto testCall = [](const std::wstring& input) {
 		constexpr wchar_t DELIM = L'$';
-		separate(input, style, name, DELIM);
+		return tokenizeFirst(input, DELIM);
 	};
 
-	std::wstring style, name;
-
 	SECTION("default case") {
-		testCall(L"foo$bar", style, name);
+		auto [style, name] = testCall(L"foo$bar");
```

Comment:
could be const (also applies to all other sections in this test)

Reply:
done